public/JavaScript/gallery.js

```diff
@@ -62,4 +62,44 @@ function reportGalleryStats (imagesInDom) {
   return String(imagesInDom)
 }
 
-module.exports = reportGalleryStats
+/**
+ *
+ * @param {Array} userSelectedFilters
+ * @returns {Object} activeFilters
```

Comment:
This says that the function returns on object.
With the recent changes, the functions does more than this.

In some cases it returns.
In others it sets activefilters object to something and then returns null.

And in others return an object.